marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -343,11 +355,7 @@ module.exports.updateJob = module.amp(
           // a better error will be thrown later when the step is run and the module cannot be found.
         }
         if (jobsReportFun) {
-          let flowStep = fn.head(datahub.hubUtils.queryLatest(function () {
-              return datahub.flow.getFlow(stepResp.flowName).steps[step];
-            },
-            datahub.config.FINALDATABASE
-          ));
+          let flowStep = getFlowStep(datahub, stepResp.flowName, step);
```

Comment:
Is there a reason to add caching here - do we know if there's a performance issue here? My understanding is that since this is only called by the mlJobs endpoint, it's only going to be called once in a given transaction, and thus the caching won't have any impact, correct?

Reply:
Removed the caching since it no longer made sense after dreverting other changes.